emerge-gitclone

```diff
@@ -20,7 +20,7 @@ with open('/usr/share/flatcar/release') as release_file:
 branch = 'flatcar-master'
 commits = {}
 if release:
-	branch = 'build-' + release.split('.', 1)[0]
+	branch = 'flatcar-build-' + release.split('+', 1)[0]
```

Comment:
Unless I'm missing something, this variable is no longer used in the code, so instead of fixing it, why not get rid of it?

Reply:
Yes, I think we can remove it if my assumption is correct.

Reply:
Done.